### ISSUE TITLE
Link handler links (t3://) not parsed anymore in Typo3 11

### ISSUE THREAD

**Issue by djangones** [oliverklee/ext-seminars#2372]
Hello.

it seems like link handler links like t3://page?uid=14 are not parsed anymore in event descriptions in the frontend with seminars 5.1.0 on Typo3 11.5.21.
They have been parsed correctly with seminars 5.1.0 on Typo3 10.4.34, thus this seems to be dependent from the used Typo3 version.

seminars: 5.1.0, Typo3 10: link handler links are parsed
seminars: 5.1.0, Typo3 11: link handler links are not parsed anymore

Unfortunately I could not figure out so far where the problem originates from, but maybe it has something to do with changes within the use/configuration of the`HtmlViewHelper::renderStatic()` helper from Typo3 10 to Typo3 11, that is used used in the `renderAsRichText()` method in the DefaultController together with the `lib.parseFunc_RTE` config?

**Comment by oliverklee** [comment]
That's my guess as well. For TYPO3 V11 (and possibly V11), we'll should try getting the real rendering context instead of the `NullRenderingContext`.

Would you be willing to create a PR for this?

**Comment by oliverklee** [comment]
Yes, sounds good. Let's do that. Looking forward to you PR! :heart:
